src/client/sandbox/node/index.js

```diff
@@ -101,4 +104,25 @@ export default class NodeSandbox extends SandboxBase {
         this.win.attach(window);
         this.element.attach(window);
     }
+
+    static _processSelector (selector) {
+        var attributeSelectorRegEx = /\[([\w-^]+)=(.+)]/;
```

Comment:
What about selector '[action=url] [href=url]'? You must use global flag  /\[([\w-^]+)=(.+)]/g